addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -302,7 +302,7 @@
 		}
 		
 		"Engineer"
-		{
+		{	
```

Comment:
Absolute horrible random extra tab out of nowhere

Reply:
fixed